extension/app/scripts.babel/background.js

```diff
@@ -21,6 +21,15 @@ const ExtensionProtocol = require('../../../helpers/extension/driver.js');
 const runner = require('../../../runner');
 const NO_SCORE_PROVIDED = '-1';
 
+window.createPageAndPopulate = function(results) {
+  const tabURL = chrome.extension.getURL('/pages/report.html');
+  chrome.tabs.create({url: tabURL}, tab => {
+    setTimeout(_ => {
+      chrome.tabs.sendMessage(tab.id, results);
+    }, 1000);
```

Comment:
I'll add a comment, but it's because though the tab exists the JS hasn't registered in the page to receive the incoming message, and so the results get lost. For more robustness I could have the page post to the bg first to ack its presence.